GCRCatalogs/photoz_magerr.py

```diff
@@ -116,7 +194,7 @@ def _iter_native_dataset(self, native_filters=None):
                 continue
             dataset = PhotoZFileObject3(file_path, self._filename_re)
             yield dataset.get
-            dataset.close() # to avoid OS complaining too many open files
+            dataset.close() # to avoid OS complaining too many open files         
```

Comment:
Remove trailing spaces
```suggestion
            dataset.close() # to avoid OS complaining too many open files
```